cli/pbjs.js

```diff
@@ -99,6 +99,8 @@ exports.main = function main(args, callback) {
                 "                   es6       ES6 wrapper (implies --es6)",
                 "                   closure   Just a closure adding to protobuf.roots (see -r)",
                 "",
+                "  --dependency     Specifies which version of protobuf to require. Accepts any valid module id",
+                "",
```

Comment:
And it will. `dependency` will be passed inside of options object to target. Target will set `dependency` to `protobufjs/minimal` if not provided externally, and will pass options to `util.wrap`.